modules/apps/portal-search/portal-search-web/src/test/java/com/liferay/portal/search/web/internal/facet/display/context/AssetTagsSearchFacetDisplayContextTest.java

```diff
@@ -181,8 +183,101 @@ public void testOneTermWithPreviousSelection() throws Exception {
 			assetTagsSearchFacetDisplayContext.isRenderNothing());
 	}
 
+	@Test
+	public void testOrderByTermFrequencyAscending() throws Exception {
+		List<TermCollector> termCollectors = _addTagsAndCreateTermCollectors(
+			new String[] {"alpha", "charlie", "bravo", "delta"},
+			new int[] {4, 5, 5, 6});
+
+		_setUpMultipleTermCollectors(termCollectors);
+
+		AssetTagsSearchFacetDisplayContext assetTagsSearchFacetDisplayContext =
+			createDisplayContext(StringPool.BLANK, "count:asc");
+
+		List<BucketDisplayContext> bucketDisplayContexts =
+			assetTagsSearchFacetDisplayContext.getBucketDisplayContexts();
+
+		String nameFrequencyString = _buildNameFrequencyString(
+			bucketDisplayContexts);
+
+		Assert.assertEquals(
+			bucketDisplayContexts.toString(),
+			"alpha:4|bravo:5|charlie:5|delta:6", nameFrequencyString);
```

Comment:
@felipelorenz That test must prove to us that the terms are ordered by Frequency Ascending, I think you must have term names as an expected result that are not ordered as Value Ascending.